test/client/utility.cc

```diff
@@ -16,6 +16,7 @@ std::unique_ptr<OptionsImpl> TestUtility::createOptionsImpl(absl::string_view ar
 }
 
 std::unique_ptr<OptionsImpl> TestUtility::createOptionsImpl(const std::vector<const char*>& argv) {
+  TCLAP::OptionalUnlabeledTracker::alreadyOptional() = false;
```

Comment:
This works around an error thrown by TCLAP about multiple unlabeled optional args not being allowed. It appeared in many tests as soon as I made URI optional. TCLAP has a global flag that detects multiple unlabeled optional args. It assumes there will be only one command line in the lifetime of the process. In unit tests we parse multiple TCLAP command lines, so we need to reset TCLAP's flag to simulate a fresh process.